app/src/main/java/edu/grinnell/grinnell_publications_android/Services/Implementation/NetworkClient.java

```diff
@@ -60,4 +60,7 @@ public Story getFullStoryById(int storyId) {
     public User getProfile() {
```

Comment:
Please don't return null! That's dangerous! If you return null and the client tries to access the return value of this method, we'll crash woefully with a `NullPointerException`